modules/client-configuration/pages/registration-methods-saltbundle.adoc

```diff
@@ -0,0 +1,33 @@
+[[registration-methods-saltbundle]]
+= Client Registration with Salt Bundle
+
+The registration method with the Salt bundle is the recommended registration method as long as it does not cover your use case.
+This section explains the advantages and limitations of the current implementation.
+The Salt bundle is provided as the [package]``venv-salt-minion`` that consists of Salt, Python, and the Python modules Salt depends on.
+Using the Salt bundle it is no longer needed that the client provides the required Python version or any Python modules.
+
+If you bootstrap new clients, registration with the Salt bundle is the default method.
+You can switch existing clients to the Salt bundle method.
+If you switch, the [package]``salt-minion`` package and its Python version and the Python modules will stay installed.
+////
+Implemented the state to switch from salt-minion to venv-salt-minion - util.mgr_switch_to_venv_minion
+
+It's better to run it in two steps:
+
+    Apply it with no pillar specified first to switch to venv-salt-minion (copy config files etc., but not cleaning up the original salt-minion confgs and packages)
+    Apply it with mgr_purge_non_venv_salt set to True to remove salt-minion and mgr_purge_non_venv_salt_files set to True to remove all the files related to salt-minion
+////
+
+On the other hand, it is also possible to avoid installing the Salt bundle and keep using [package]``salt-minion`` instead.
+In this case specify one of these options:
+
+* Execute [command]``mgr-bootstrap`` with  [option]``--no-bundle`` option.
+* Set [literal]``AVOID_VENV_SALT_MINION`` to [literal]``1`` in the generated bootstrap script.
+* For bootstrap state set the [literal]``mgr_avoid_venv_salt_minion`` pillar set to [literal]``True``.
+
+[IMPORTANT]
+====
+The Salt bundle does not support the Salt SSH contact method.
+Stay with [package]``salt-minion`` package if you use Salt SSH.
+====
+
```

Comment:
Bootstrapping client with Salt-SSH contact method is not using salt-minion, but the salt codebase is deployed with salt-thin tarball with no installation any additional packages. It's better just remove this IMPORTANT section.

Reply:
ok, I'll submit this removal separately.  locally committed.